android/app/src/main/java/com/expensify/chat/intentHandler/AbstractIntentHandler.kt

```diff
@@ -0,0 +1,19 @@
+package com.expensify.chat.intentHandler
```

Comment:
Minor thing but Java package names are typically all lowercase 😄 

```suggestion
package com.expensify.chat.intenthandler
```